grc/core/blocks/embedded_python.py

```diff
@@ -230,18 +229,17 @@ def double(arg):
         to set parameters of other blocks in your flowgraph.
     """)}
 
-    parameters = [dict(
-        label='Code',
-        id='source_code',
-        dtype='_multiline_python_external',
-        value='# this module will be imported in the into your flowgraph',
-        hide='part',
-    )]
+    parameters = build_params(
```

Comment:
This is supposed to be `parameters_data`, no? Same with the virtual blocks

Reply:
your absolutely right. good catch. fixed. note to self, stop rebasing after manual testing =)